### DISCUSSION TITLE
关于下载时全部报错的求助

### DISCUSSION THREAD

**Discussion by Frogjiangs** [the1812/Bilibili-Evolved#1462] [提问 / Q&A]
是新鲜下载的ef2文件，也有大会员等等，下载着下载着突然就不下了，然后就全都“错误”了
![FAI@PYXC%Y3C 57Q@9RXV7X](https://user-images.githubusercontent.com/77086332/104400092-f179b600-558c-11eb-9e52-6b3268ea2ff2.png)

**Comment by Frogjiangs** [comment]
麻烦大家了，谢谢！

**Comment by the1812** [comment]
最近发现链接导出来过一会偶尔会挂掉, 重新导一次就行了
